pytest_operator/plugin.py

```diff
@@ -329,18 +329,20 @@ async def build_charm(self, charm_path):
         if layer_path.exists():
             # Handle older, reactive framework charms.
             check_deps("charm")
-            cmd = ["charm", "build", "-F", charm_abs]
+            cmd = ["charm", "build", "-F"]
```

Comment:
Does this work?  I think this still needs to be either `charm_abs` or
```suggestion
            cmd = ["charm", "build", "-F", "./"]
```

Reply:
From `charm build --help`:

```
positional arguments:
  charm                 Source directory for charm layer to build (default: .)
```

Reply:
mixed up charm and charmcraft.  whoops!